dracut/31decrypt-root/decrypt-root

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -euo pipefail
```
Would be good to enable error handling. The below line would need to change a bit, though, but in most cases it helps to discover unwanted behavior.